lib/views/pages/create_account_page.dart

```diff
@@ -0,0 +1,69 @@
+import "package:flutter/material.dart";
+import "package:hooks_riverpod/hooks_riverpod.dart";
+import "package:proxima/viewmodels/login_view_model.dart";
+import "package:proxima/views/navigation/routes.dart";
+
+class CreateAccountPage extends HookConsumerWidget {
+  static const logoutButtonKey = Key("logout");
+  static const confirmButtonKey = Key("confirm");
+
```

Comment:
Nice implementation ! Especially with the resizable interface when the keyboard is opened which makes the app more responsive.
 
Little suggestion, maybe we could add keys for the `unique username` and `pseudo` fields so that we can check that they are displayed in the tests.

```suggestion

  static const uniqueUsernameFieldKey = Key("uniqueUsername");
  static const pseudoFieldKey = Key("pseudo");
```